openstack/arc/charts/postgresql/values.yaml

```diff
@@ -29,43 +29,10 @@ persistence:
   # Re-use existing (unmanged) PVC
   # existingClaim: claimName
 
-metrics:
-  enabled: false
-  image: "wrouesnel/postgres_exporter"
-  imageTag: v0.8.0
-  imagePullPolicy: IfNotPresent
-  customMetrics:
-    pg_database:
-      query: "SELECT d.datname AS datname, CASE WHEN pg_catalog.has_database_privilege(d.datname, 'CONNECT') THEN pg_catalog.pg_database_size(d.datname) ELSE 0 END AS size_bytes FROM pg_catalog.pg_database d where datname not in ('template0', 'template1', 'postgres')"
-      metrics:
-        - datname:
-            usage: "LABEL"
-            description: "Name of the database"
-        - size_bytes:
-            usage: "GAUGE"
-            description: "Size of the database in bytes"
-  resources:
-    requests:
-      memory: 256Mi
-      cpu: 100m
-
 ## Configure resource requests and limits
 ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
 ##
 resources:
   requests:
     memory: 256Mi
     cpu: 100m
-
-backup:
-  enabled: false
-  repository: sapcc/backup-tools
-  image_version: v0.6.5
```

Comment:
0.7.0. This is now hardcoded in the Helm chart because being able to configure it is useless in practice: https://github.com/sapcc/helm-charts/blob/master/common/pgbackup/templates/deployment.yaml#L36 - Future updates will be handled by bumping the subchart version.